function.peer.new.php

```diff
@@ -6,21 +6,33 @@ function peer_new() {
 
 	require_once __DIR__.'/once.db.connect.php';
 
+	$compactv4 = '';
+	$compactv6 = '';
+	if ( isset($_GET['ipv4'])) {
+		$compactv4 = bin2hex(pack('Nn', ip2long($_GET['ipv4']), $_GET['portv4']));
+	}
+	if ( isset($_GET['ipv6'])) {
+		$compactv6 = bin2hex( inet_pton($_GET['ipv6']) . pack('n', $_GET['portv6']) );
```

Comment:
Inconsistent style, should be: `$compactv6 = bin2hex(inet_pton($_GET['ipv6']).pack('n', $_GET['portv6']));`